plugins/catalog-backend-module-azure/src/lib/azure.ts

```diff
@@ -47,7 +50,7 @@ export async function codeSearch(
   const searchBaseUrl = isCloud(azureConfig.host)
     ? 'https://almsearch.dev.azure.com'
     : `https://${azureConfig.host}`;
-  const searchUrl = `${searchBaseUrl}/${org}/${project}/_apis/search/codesearchresults?api-version=6.0-preview.1`;
+  const searchUrl = `${searchBaseUrl}/${org}/_apis/search/codesearchresults?api-version=6.0-preview.1`;
```

Comment:
I'm a bit worried this might have an impact on auth, in the case where one has credentials that only has access to the project, if that's possible?

Perhaps it's best to keep using the existing URL for cases where we're looking for a specific project?

Reply:
Mm good point but i wander if that is true.

The credential used here is a Personal Access Token(PAT)[see the docs](https://backstage.io/docs/integrations/azure/locations), so if in that regard if you don't have access to a specific project inside the organization, it is not listed because the PAT impersonates is scoped to you as a person.

Inside the PAT there are no option to specify the Project that you allow or disallow as that is inherited by what you as a user can see. The rights that you can set are for these and are more technical of nature rather then tight to projects:

![image](https://user-images.githubusercontent.com/676025/204788591-c1959dad-f36b-4d7b-9a55-13be01843f18.png)

Reply:
This shouldn't cause issues since the PAT is scoped to the org. It should filter to only the projects the PAT has access to. The example on the docs is using the org url.

https://learn.microsoft.com/en-us/rest/api/azure/devops/search/code-search-results/fetch-code-search-results?view=azure-devops-rest-7.0&tabs=HTTP#examples